src/routes.js

```diff
@@ -17,7 +19,8 @@ export default () => (
         <Route exact path="/" component={Home} />
         <Route exact path="/signin" component={Signin} />
         <Route exact path="/login" component={SocialAuth} />
-
+        <Route exact path="/reset/password" component={ForgotPasssword} />
```

Comment:
Take advantage of the new file that contains Routes by Korir.
Include all routes in a central point